Integrations/FeedTAXII/FeedTAXII.py

```diff
@@ -896,6 +931,11 @@ def interval_in_sec(val):
 
 
 def test_module(client, args):
+    all_collections = client.get_all_collections()
+    if client.collection not in all_collections:
```

Comment:
Please add another level of error handling here - my main concern here is if there's a problem with the `Collection Management Service`, while the user provided a valid collection, he'll get an error though his instance is usable.
This is a valid scenario since all services are optional. See http://taxiiproject.github.io/releases/1.1/TAXII_Services_Specification.pdf

Reply:
According to the specification it seems like we can't really know if the list we are receiving is incomplete. If the issue was that we get an error then we could handle that, but an incomplete list is an issue that we can't really handle.
So, maybe a better error message will do the trick?